app/models/alert_types/wiki_expert_notification_alert.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: alerts
+#
+#  id             :integer          not null, primary key
+#  course_id      :integer
+#  user_id        :integer
+#  article_id     :integer
+#  revision_id    :integer
+#  type           :string(255)
+#  email_sent_at  :datetime
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#  message        :text(65535)
+#  target_user_id :integer
+#  subject_id     :integer
+#  resolved       :boolean          default(FALSE)
+#  details        :text(65535)
+
+class WikiExpertNotificationAlert < Alert
```

Comment:
I should have noticed this in my first pass, but... this is not a good name for this alert. It should be specific to the situation it's designed around. I suggest EarlyEnrollmentAlert (and a similar name for the mailer).